content/patterns/disclosure/examples/disclosure-navigation-hybrid.html

```diff
@@ -205,22 +198,22 @@ <h2 id="kbd_label">Keyboard Support</h2>
                 <kbd>Tab</kbd><br>
                 <kbd>Shift + Tab</kbd>
               </th>
-              <td>Move keyboard focus among top-level links and buttons, and if a dropdown is open, through links in the dropdown.</td>
+              <td>トップレベルのリンクとボタン間、及びドロップダウンが開いている場合は、ドロップダウン内のリンクを通じてキーボードフォーカスを移動する。</td>
```

Comment:
「ドロップダウン内のリンクを通じて」という訳が、若干わかりにくいかなと感じました。through のニュアンスですが、ここでは「〜を通して」というのはいかがでしょうか？

```suggestion
              <td>トップレベルのリンクとボタン間、及びドロップダウンが開いている場合は、ドロップダウン内のリンクを通してキーボードフォーカスを移動する。</td>
```

Reply:
「, and if ...」という繋ぎかたがあまり日本語になじまないなあという感じもあり、2文に分けたうえ、述語部分を最初の文のほうに持ってきてみました。「通して」も反映しました。ニュアンスがむずかしいですね・・

```suggestion
              <td>トップレベルのリンクとボタン間をキーボードフォーカスを移動させる。またドロップダウンが開いている場合は、ドロップダウン内のリンクを通して移動させる。</td>
```

どうでしょうか・・？